framework/arcane-framework/src/test/scala/models/CdmTableTests.scala

```diff
@@ -0,0 +1,42 @@
+package com.sneaksanddata.arcane.framework
+package models
+
+import models.cdm.SimpleCdmModel
+import services.storage.models.azure.AzureBlobStorageReader
+
+import com.azure.storage.common.StorageSharedKeyCredential
+import services.cdm.{CdmTable, CdmTableSettings}
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.matchers.should.Matchers.should
+
+import java.time.{OffsetDateTime, ZoneOffset}
+
+
+class CdmTableTests extends AsyncFlatSpec with Matchers {
```

Comment:
Did I get it correct that this test should be executed manually against an existing storage container? If yes, would be nice to mention it in the class javadoc

Reply:
only the first two tests. I'll add tests that do not rely on Azure but rather test read model + read csv functionality instead